extract_data.py

```diff
@@ -20,48 +20,13 @@ def extract_and_collect_data_from_generated_files(directory: str, has_dedicated_
 
     chassis, mobo, cpu, dimms, gpu, disks, psu = extract_data(directory, has_dedicated_gpu, gpu_in_cpu,
                                                               cleanup=cleanup, verbose=verbose, unpack=False)
-    no_dimms_str = "decode-dimms was not able to find any RAM details"
 
     # the None check MUST come before the others
-    """if dimms.__len__() == 0:
-        # empty default dictionary
-        dimms = {
-            "type": "ram",
-            "working": "yes",
-            "brand": None,
-            "model": None,
-            "serial_number": None,
-            "frequency": None,
-            "human_readable_frequency": None,
-            "capacity": None,
-            "human_readable_capacity": None,
-            "RAM_type": None,
-            "ECC": None
-            # "CAS_latencies": None # feature missing from TARALLO
-        }"""
 
     no_gpu_info_str = "I couldn't find the Video Card brand. The model was set to 'None' and is to be edited logging " \
                       "into the TARALLO afterwards. The information you're looking for should be in the following 2 lines:"
     no_vram_info_str = "A dedicated video memory couldn't be found. A generic video memory capacity was found instead, which " \
                        "could be near the actual value. Please humans, fix this error by hand."
-
-    """if gpu.__len__() == 0 or (no_gpu_info_str in gpu and no_vram_info_str in gpu):
-        gpu = {
-            "type": "graphics-card",
-            "manufacturer_brand": None,
-            "reseller_brand": None,
-            "model": None,
-            "capacity": None,
-            "human_readable_capacity": None
-        }
-        print_lspci_lines_in_dialog = True
-
-    elif no_vram_info_str in gpu and no_gpu_info_str not in gpu:
-        # TODO: check output in this case, change only VRAM field to None
-        print_lspci_lines_in_dialog = False
-    else:
-        print_lspci_lines_in_dialog = False
-    """
     if chassis.__len__() == 0:
```

Comment:
[formal] please refactor **everywhere** as follows:
```suggestion
    if len(chassis) == 0:
```